packages/aws-cdk/.gitignore

```diff
@@ -24,3 +24,11 @@ npm-shrinkwrap.json
 test/integ/cli-backwards-tests-*
 !.eslintrc.js
 !jest.config.js
+
+# Might be created when running integ tests
+cdk.context.json
+
+# Exclude compiled integ tests specifically (but not any subdir
+# as the subdirs contain .js files that should be committed)
```

Comment:
Curious: why do we have `.js` files? legacy or intentional?

Reply:
A quick `find` command revealed

```
./test/integ/bootstrap/example-cdk-app/my-test-cdk-stack.js
./test/integ/cli/app/lambda/index.js
./test/integ/cli/app/nested-stack.js
./test/integ/cli/app/app.js
./test/integ/cli/cloud-assemblies/1.10.0-lookup-default-vpc/manifest.json.js
./test/integ/cli/cloud-assemblies/1.10.0-request-azs/manifest.json.js
./test/integ/cli.exclusions.js
./.eslintrc.js
./lib/init-templates/sample-app/typescript/jest.config.js
./lib/init-templates/sample-app/javascript/test/%name%.test.template.js
./lib/init-templates/sample-app/javascript/bin/%name%.template.js
./lib/init-templates/sample-app/javascript/lib/%name%-stack.template.js
./lib/init-templates/app/typescript/jest.config.js
./lib/init-templates/app/javascript/test/%name%.test.template.js
./lib/init-templates/app/javascript/bin/%name%.template.js
./lib/init-templates/app/javascript/lib/%name%-stack.template.js
./lib/init-templates/lib/typescript/jest.config.js
```

Reply:
Some of them definitely intentional (init templates, `something.confi.js`), other ones because of convenience (well I need templating of JSON and a js file+Node can give me that easily)